### ISSUE TITLE
Problem with Chrome minimum font-size

### ISSUE THREAD

**Issue by Explorare** [ilyalissoboi/FuriganaInjectorPlusPlus#2]
The default font-size in Chrome depends on minimum font-size. The default value is 12px. And mine is 14px for a 15.5 inch 1920*1080 display. The Furigana will be rendered to 10px in the following example.

![2016-12-05-185353-001](https://cloud.githubusercontent.com/assets/5090766/20882605/c557f62c-bb1c-11e6-81c3-1122eb57e8fb.png)

![2016-12-05-185332-001](https://cloud.githubusercontent.com/assets/5090766/20882611/ce280e22-bb1c-11e6-860b-7f82887b4615.png)

The CSS shown the Furiganas are 6px, but in fact they are 10px.

The sample in the settings won't be smaller below 14px, cause I've set the minimum font-size to 14px. If I set it to 10px or smaller, all the fonts will use that as the default value and become black dots.

![2016-12-05-185638-001](https://cloud.githubusercontent.com/assets/5090766/20882783/a28efd7e-bb1d-11e6-9868-3e48be66e10f.png)

![2016-12-05-185450-001](https://cloud.githubusercontent.com/assets/5090766/20882776/9b4cb3b2-bb1d-11e6-8fd3-c267418d4746.png)

But it seems that the Furiganas in that page are rendered into 10px. How did that happen? Shouldn't it be 14px?

It there a way to fix this?

**Comment by ilyalissoboi** [comment]
Hm, can you explain the exact issue you're having a bit more? Do you want the kana to be rendered at the exact size specified or at the minimum font size set in Chrome settings?

**Comment by Explorare** [comment]
Yes. They should be 6px as CSS values. But in fact they are rendered as 10px.

**Comment by ilyalissoboi** [comment]
I'll look into that, but it might be a browser limitation, since the minimum font size is meant to override CSS settings, AFAIK.

**Comment by Explorare** [comment]
But the minimum font-size is 14px on my Chrome. Why the kana are rendered into 10px? So it can be rendered smaller than minimum font-size, then they can be rendered smaller than 10px?

**Comment by ilyalissoboi** [comment]
I don't know. Maybe ruby tags have some special interactions with these settings. I've never tried playing around with the minimum font size, which is why I need to do some research, before I can really tell you anything.

**Comment by Explorare** [comment]
[Chromeでfont-sizeを10px以下に設定する方法 - 世界の一部](https://time-complexity.blogspot.jp/2013/12/chrome-font-size-10px.html) 
This will fix the problem.
`-webkit-transform: scale(0.75)`
